.github/workflows/test-build.yml

```diff
@@ -0,0 +1,52 @@
+name: Test action build
+
+on:
+  workflow_call:
+
+jobs:
+  test-build:
+    name: Check if the source files build successfully and match dist/ directory
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the action
+        uses: actions/checkout@v3
+      - name: Setup Node.js 16.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install npm-cli-login
+        run: npm install -g npm-cli-login
+      - name: Login to GitHub npm registry
+        run: |
+          npm-cli-login -s @actions-rs -r https://npm.pkg.github.com \
+            -u $ -p ${{ secrets.GITHUB_TOKEN }} -e test@example.com
```

Comment:
Is the `-u $` intended here? It seems like a strange choice of syntax since the `$` is a special character both for GitHub Actions and for the shell 🤔

Reply:
You're right, it is definitely not the best placeholder... I changed it a little and added a comment, please have a look now ;)